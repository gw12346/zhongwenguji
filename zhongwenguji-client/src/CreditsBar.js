import React from 'react';
import style from './CreditsBar.module.scss';

// eslint-disable-next-line jsx-a11y/anchor-has-content
const Link = (props) => (<a target="_blank" rel="noopener" {...props} />);

export default () => (
  <div className={style.container}>
    <span>Non-linear regression code by <Link href="https://github.com/jonasalmeida/fminsearch">Dr. Jonas Almeida</Link></span>
    <span>Chinese character frequency list by <Link href="http://lingua.mtsu.edu/chinese-computing/statistics/char/list.php?Which=MO">Dr. Jun Da</Link></span>
    <span>Icons by <Link href="https://www.flaticon.com/authors/egor-rumyantsev">Egor Rumyantsev</Link>, <Link href="https://www.flaticon.com/authors/dave-gandy">Dave Gandy</Link> and <Link href="https://www.freepik.com/">Freepik</Link>
    </span>
    <span>Created with love by <Link className={style.selfLink} href="https://www.linkedin.com/in/codyjenkins/">iknowcss</Link></span>
  </div>
);
