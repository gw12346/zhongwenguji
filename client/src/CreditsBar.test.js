import React from 'react';
import Renderer from 'react-test-renderer';
import CreditsBar from './CreditsBar';

describe('CreditsBar', () => {
  it('renders', () => {
    const component = Renderer.create(<CreditsBar />);
    expect(component).toMatchSnapshot();
  });
});
