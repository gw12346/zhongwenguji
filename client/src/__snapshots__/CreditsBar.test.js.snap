// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditsBar renders 1`] = `
<div
  className="container"
>
  <span>
    Non-linear regression code by 
    <a
      href="https://github.com/jonasalmeida/fminsearch"
      rel="noopener"
      target="_blank"
    >
      Dr. Jonas Almeida
    </a>
  </span>
  <span>
    Chinese character frequency list by 
    <a
      href="http://lingua.mtsu.edu/chinese-computing/statistics/char/list.php?Which=MO"
      rel="noopener"
      target="_blank"
    >
      Dr. Jun Da
    </a>
  </span>
  <span>
    Icons by 
    <a
      href="https://www.flaticon.com/authors/egor-rumyantsev"
      rel="noopener"
      target="_blank"
    >
      Egor Rumyantsev
    </a>
    , 
    <a
      href="https://www.flaticon.com/authors/dave-gandy"
      rel="noopener"
      target="_blank"
    >
      Dave Gandy
    </a>
     and 
    <a
      href="https://www.freepik.com/"
      rel="noopener"
      target="_blank"
    >
      Freepik
    </a>
  </span>
  <span>
    Created with love by 
    <a
      className="selfLink"
      href="https://www.linkedin.com/in/codyjenkins/"
      rel="noopener"
      target="_blank"
    >
      iknowcss
    </a>
  </span>
</div>
`;
